frontend/src/metabase/search/analytics.ts

```diff
@@ -0,0 +1,35 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+import type { URLSearchFilterQueryParams } from "metabase/search/types";
+
+export const trackSearchEvents = ({
+  searchFilters,
+}: {
+  searchFilters: URLSearchFilterQueryParams;
+}) => {
+  let content_type = null;
+
+  if (searchFilters.type) {
+    content_type = Array.isArray(searchFilters.type)
+      ? searchFilters.type
+      : [searchFilters.type];
+  }
+
+  const filterBooleanPayload: { [key: string]: unknown } = {
```

Comment:
The rest of these variables are booleans so we can just run the same `!!` operation on all of them.

Reply:
`Record` usage is more established. So I suggest we follow that.

```suggestion
  const filterBooleanPayload: Record<string, unknown> = {
```